Bloxstrap/App.xaml.cs

```diff
@@ -227,6 +237,25 @@ protected override void OnStartup(StartupEventArgs e)
 
                     commandLine = $"--app --deeplink {LaunchArgs[0]}";
                 }
+                else if (LaunchArgs[0].StartsWith("roblox-studio:"))
+                {
+                    commandLine = ProtocolHandler.ParseUri(LaunchArgs[0]);
+                    if (!commandLine.Contains("-startEvent"))
+                        commandLine += " -startEvent www.roblox.com/robloxQTStudioStartedEvent";
+                    isStudioLaunch = true;
+                }
+                else if (LaunchArgs[0].StartsWith("roblox-studio-auth:"))
+                {
+                    commandLine = HttpUtility.UrlDecode(LaunchArgs[0]);
+                    isStudioLaunch = true;
+                    isStudioAuth = true;
+                }
+                else if (LaunchArgs[0] == "-ide")
+                {
+                    isStudioLaunch = true;
+                    if (LaunchArgs.Length >= 2)
+                        commandLine = $"-task EditFile -localPlaceFile \"{LaunchArgs[1]}\"";
+                }

```

Comment:
You can simply pass Studio `-protocolString`, the same can be said for player by simply providing the launch URI, as player and studio can handle the URI independently!